libs/ledger-live-common/src/families/evm/walletApiAdapter.test.ts

```diff
@@ -18,7 +19,7 @@ describe("getWalletAPITransactionSignFlowInfos", () => {
         recipient: ethPlatformTx.recipient,
         data: undefined,
         gasLimit: undefined,
-        nonce: undefined,
```

Comment:
I still don't understand how this test ever worked typescript wise as `nonce` had a `number` type for at least the last 16 months based on a git blame 🤔

Reply:
It's because of the `Partial<Transaction>` in line 16 so every value can be of the expected type or `undefiend`